tests/src/Context/FieldContext.php

```diff
@@ -141,7 +141,7 @@ private function scanFieldGroupsForResetting($field_groups_def_level, $default_d
 
         default:
           if (!$current_default_def_level) {
-            field_group_group_export_delete($item, FALSE);
```

Comment:
According with the documentation (http://api.drupalhelp.net/api/field_group/field_group.module/function/field_group_group_export_delete/7.2), the default value of the second parameter "$ctools_crud" is TRUE. so it is not necessary to put it here.